exporters/elasticsearch/src/es_log_exporter.cc

```diff
@@ -3,8 +3,11 @@
 
 #ifdef ENABLE_LOGS_PREVIEW
 
+#  include <sstream>  // std::stringstream
+
 #  include "opentelemetry/exporters/elasticsearch/es_log_exporter.h"
 #  include "opentelemetry/exporters/elasticsearch/es_log_recordable.h"
+#  include "opentelemetry/sdk/common/global_error_handler.h"
```

Comment:
You may append the standard set of logging macros somewhere once in `api/include/opentelemetry/config.h` - to avoid explicitly including the global error handler in each exporter.